examples/network/testchain.json

```diff
@@ -27,7 +27,7 @@
         "0xf4adb4c4f708c4b753657373696f6e204e6578744b657973343a73657373696f6e3a6b657973e54094c2d5af8ae10b91e1288f4f59f2946d7738f2c509b7effd909e5e9ba0ad": "0x02a4e017293b86f3b74df2f3b033a9de02b6e5e36e7ca1b643c92eefb5cdc09d469e241f66c7d748c399174eb5026b0e2cfa5e66f51755f13a4e6a89c6788228469e241f66c7d748c399174eb5026b0e2cfa5e66f51755f13a4e6a89c6788228",
         "0xf4adb4c4f708c4b753657373696f6e204e6578744b657973343a73657373696f6e3a6b657973711590f60a214f6f06502eb29dd14f55aa04e72e2fa12c098ba4fa5a00c57fa9": "0xba891366d247202ef7f88e81d49ef4c62bc5d23173e6b5fb0bca2817d856c980ec7e5a695418ce05b7d1151693027e9a322973cba86477fea8db432a0a331261ec7e5a695418ce05b7d1151693027e9a322973cba86477fea8db432a0a331261",
         "0x71020fee971bd00e8248d1830b8cffbe5b9cf4de1ea2911a1665c44fd70ab6f3": "0xf26cdb14b5aec7b2789fd5ca80f979cef3761897ae1f37ffb3e154cbcc1c26630f0000c16ff286230f0000c16ff2862300",
-        "0x3a6772616e6470615f617574686f726974696573": "0x011003d80edbdfb0316668dd57a530479ef79138e67a2e6f0cf545ec51fe30ea66f0010000000000000002a4e017293b86f3b74df2f3b033a9de02b6e5e36e7ca1b643c92eefb5cdc09d0100000000000000ba891366d247202ef7f88e81d49ef4c62bc5d23173e6b5fb0bca2817d856c980010000000000000056711b24c9ff377a11e515f09d5504abfa7247c964d55700893363e8bd54bf940100000000000000",
+        "0x3a6772616e6470615f617574686f726974696573": "0x011003d80edbdfb0316668dd57a530479ef79138e67a2e6f0cf545ec51fe30ea66f0010000000000000002a4e017293b86f3b74df2f3b033a9de02b6e5e36e7ca1b643c92eefb5cdc09d0100000000000000ba891366d247202ef7f88e81d49ef4c62bc5d23173e6b5fb0bca2817d856c9800100000000000000d046dde66d247e98e6c95366c05b6137ffeb61e9ee8541200569e70ac7632a460100000000000000",
```

Comment:
What happened here?

Reply:
It fixes wrong authorities over replace genesis config.